packages/cli/src/cmds/validator/options.ts

```diff
@@ -237,7 +237,7 @@ export const validatorOptions: CliCommandOptions<IValidatorCliArgs> = {
 
   "builder.selection": {
     type: "string",
-    description: "Default builder block selection strategy: maxprofit or builderalways",
+    description: "Default builder block selection strategy: maxprofit or builderalways or builderonly",
```

Comment:
Might be easier to read like this
```suggestion
    description: "Default builder block selection strategy: maxprofit, builderalways, or builderonly",
```